product/runtime/src/main/python/java/android/importer.py

```diff
@@ -528,8 +532,11 @@ def extract_so(self, path):
 
     def extract_dir(self, zip_dir, recursive=True):
         dotted_dir = zip_dir.replace("/", ".")
-        extract_package = any((dotted_dir == ep) or dotted_dir.startswith(ep + ".")
-                              for ep in self.extract_packages)
+        if "*" in self.extract_packages:
+            extract_package = True
```

Comment:
This could be merged into the `any` expression, e.g. `extract_package = any((ep == "*") or (dotted_dir == ep) or ...`.

Reply:
Also, please follow either the original indentation style, or Black.